internal/services/postgres/postgresql_server_resource.go

```diff
@@ -376,6 +374,22 @@ func resourcePostgreSQLServer() *pluginsdk.Resource {
 			}),
 		),
 	}
+
+	if !features.FivePointOh() {
+		resource.Schema["ssl_minimal_tls_version_enforced"] = &pluginsdk.Schema{
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			Default:      string(servers.MinimalTlsVersionEnumTLSOneTwo),
+			ValidateFunc: validation.StringInSlice(servers.PossibleValuesForMinimalTlsVersionEnum(), false),
+		}
+		resource.Schema["ssl_enforcement_enabled"] = &pluginsdk.Schema{
+			Deprecated: "The `ssl_enforcement_enabled` property is deprecated as Azure services will require TLS1.2+ to connect.",
```

Comment:
We should add the major version in which this property will be removed

```suggestion
			Deprecated: "The `ssl_enforcement_enabled` property is deprecated and will be removed in v5.0 of the AzureRM provider as Azure services will require TLS1.2+ to connect.",
```